code/lib/preview-web/src/render/StoryRender.ts

```diff
@@ -242,6 +242,7 @@ export class StoryRender<TFramework extends AnyFramework> implements Render<TFra
           await this.runPhase(abortSignal, 'errored', async () => {
             this.channel.emit(PLAY_FUNCTION_THREW_EXCEPTION, serializeError(error));
           });
+          if (!this.story.parameters.hidePlayFunctionExceptions) throw error;
```

Comment:
Maybe invert it and rename to `throwPlayFunctionExceptions`. Seems more descriptive (and in general, "negated" naming is confusing).

Reply:
I don't want to invert it because the default is for it to be unset.

Reply:
Naming is important and hard to change later down the road. I'd rather have to "complicate" our logic a bit than to introduce a parameter name that could've been named better.